docker/compose/postgres/model/07-clean-up-blog-objects.psql

```diff
@@ -0,0 +1,37 @@
+SET search_path TO EVENTS;
+
+-- ensure an object for the '/stories' uri exists for all /blog uris
+
+INSERT INTO objects (uri)
+  (SELECT DISTINCT REPLACE(uri, '/blog/', '/stories/')
+   FROM objects) ON CONFLICT DO NOTHING;
+
+-- on actions/history linked to /blog/ objects, update the link to be to the /stories/ object
+
+UPDATE actions
+SET object_id=good_id
+FROM
+  (SELECT bad.id bad_id,
```

Comment:
These are not the result of a bug, so let's call them old and new instead of bad and good.